src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -160,6 +160,7 @@ void VtolAttitudeControl::vehicle_cmd_poll()
 
 			} else {
 				_transition_command = int(vehicle_command.param1 + 0.5f);
+				_immediate_transition = int(vehicle_command.param2 + 0.5f);
```

Comment:
I wasn't 100% if this is the correct way to do it

Reply:
I also don't get the reasoning behind the +0.5, but prob okay to just do the same as for the transition command for now.

Reply:
I think it's to make sure that, as the command.param values are sent as float over mavlink you might be unlucky and a 1 gets sent as 0.99999-something, which could then be int-converted to 0. Whereas if you are at 0.99999+0.5f you make sure to get the correct conversion.

For me it's more that I'm adding yet another conversion float -> int -> bool, so I'm not sure if this is smartest way. Anyway, it works in my tests, so I guess it's fine. :)

Reply:
Makes sense :+1: